### PR TITLE
setting userAuthUserId tag from ai_authUser cookie field

### PR DESCRIPTION
**Pull request by spike008t**
A small contribution to populate `userAuthUserId`  from `ai_authUser` cookie field (like `ai_user`)

Seems that also concerns #356

### CODE REVIEW

**Comment by OsvaldoRosado** [comment]
Thanks for this PR! Looks great!

File: Tests/AutoCollection/HttpRequestParser.tests.ts [hunk 1]
```diff
@@ -122,6 +123,7 @@ describe("AutoCollection/HttpRequestParser", () => {
             var newTags = helper.getRequestTags(originalTags);
             assert.equal(newTags[(<any>HttpRequestParser).keys.locationIp], 'originalIp');
             assert.equal(newTags[(<any>HttpRequestParser).keys.userId], 'originalUserId');
+            assert.equal(newTags[(<any>HttpRequestParser).keys.userAuthUserId], 'originalAuthUserId');
```
Comment (OsvaldoRosado) [inline Tests/AutoCollection/HttpRequestParser.tests.ts]: You might want to add another assertion like this to the next section "should read tags from headers"

Comment (spike008t) [reply Tests/AutoCollection/HttpRequestParser.tests.ts]: Yep! I've didn't notice the next section.
Done!

Comment (spike008t) [reply Tests/AutoCollection/HttpRequestParser.tests.ts]: @OsvaldoRosado I still dont know why the tests failed...

Comment (OsvaldoRosado) [reply Tests/AutoCollection/HttpRequestParser.tests.ts]: Looks like something changed in the test environment. I'll look into this

**Pull request merged**